### DISCUSSION TITLE
Get The DWG drawing Layer data

### DISCUSSION THREAD

**Discussion by Antika-Chowdhury** [DomCR/ACadSharp#334] [Q&A]
Hello Team,

As per current solution only layer names are visible. I want to fetch the layer details like data building height or width I want to fetch that.  How can I do that?

**Comment by DomCR** [comment]
Hi @Antika-Chowdhury,

I'm guessing that what you are looking for is the geometric information of the file, the Layers doesn't contain any of that, they are only records in a table to organize the different entities in the drawing.

If you want to get the entities you can use the `CadDocument.Entities` property which is a list with all the entities drawn in the cad file.

For more information of how a dxf/dwg file are organized you can check this link from the documentation, `ACadSharp` uses a similar organization:

https://help.autodesk.com/view/OARX/2021/ENU/?guid=GUID-A809CD71-4655-44E2-B674-1FE200B9FE30

Let me know if this helps.
